bin/parser.js

```diff
@@ -48,7 +48,8 @@ if (!program.input) {
         rdf: namespaces.rdf,
         rdfs: namespaces.rdfs,
         fnml: "http://semweb.mmlab.be/ns/fnml#",
-        fno: "http://w3id.org/function/ontology#"
+        fno: "http://w3id.org/function/ontology#",
```

Comment:
Whilst we're at it, maybe change to https?

Reply:
Done. Can you check if it's fine?